README.md

```diff
@@ -16,6 +16,8 @@ that will send logs to the [Logstash UDP input](https://www.elastic.co/guide/en/
  be merged with the metadata sent in every log message.
  * **level**: Atom. Minimum level for this backend.
  * **type**: String.t. Type of logs. Useful to filter in logstash.
+ * **timezone**: String.t. Server timezone. Used to convert from naive timestamp. Default `"Etc/UTC"`.
+ * **json_encoder**: Atom. Module to be used for JSON encoding. Default `Jason`.
```

Comment:
Since `Jason` is only in the `:dev` environment, should we add a note to include it in dependency list if they haven't already?

Reply:
Good point. I've also removed it from the `dev` dependencies (so now it's only in `test`) so that nobody deploys a broken system to production by accident.